timeutil/schedule.go

```diff
@@ -152,13 +330,20 @@ var weekdayMap = map[string]int{
 	"sat": 6,
 }
 
+var weekdayOrder = []string{
+	"mon",
+	"tue",
+	"wed",
+	"thu",
+	"fri",
+	"sat",
+	"sun",
+}
+
 // parseTimeInterval gets an input like "9:00-11:00"
 // and extracts the start and end of that schedule string and
 // returns them and any errors.
 func parseTimeInterval(s string) (start, end TimeOfDay, err error) {
-	if strings.Contains(s, "@") {
```

Comment:
The changes here are slightly worrying. I'd prefer to not touch that legacy code, so we're sure the original algorithm still behaves as it should, when parsing the old specification.

Reply:
#4270 dropped support for parsing weekday in the 'old' parser. I believe this was just left behind accidentally.